src/p11_child/p11_child_common.c

```diff
@@ -101,11 +102,14 @@ static int do_work(TALLOC_CTX *mem_ctx, enum op_mode mode, const char *ca_db,
     return ret;
 }
 
-static errno_t p11c_recv_data(TALLOC_CTX *mem_ctx, int fd, char **pin)
+static errno_t p11c_recv_data(TALLOC_CTX *mem_ctx, int fd, char **pin,
+                              uint64_t *_old_chain_id)
 {
     uint8_t buf[IN_BUF_SIZE];
     ssize_t len;
+    size_t p = 0;
```

Comment:
What is the purpose of `p`?

Reply:
This code is now gone in the latest update, so please disregard.